sdk-wallet/user_signer.md

```diff
@@ -0,0 +1,14 @@
+## UserSigner
+
+```
+class UserSigner:
+    constructor(secret_key: UserSecretKey);
+
+    // Named constructors
+    static fromPemFile(path: Path, index: number = 0): UserSigner;
```

Comment:
Let's not mix again private/public key definitions or loading with actual signing/verifying.
Why not use 
```
sign(privateKey UserSecretKey, data: bytes): bytes;
```
same for the UserVerifier:
```
verify(publicKey UserPublicKey, data: bytes, signature: bytes): boolean;
```

Now signer & verifier can be merged into a single component. Then, aggregated into a larger one called CryptoProvider

Reply:
Fixed in the separate PR for sdk-wallet, now brought here.